.eslintrc

```diff
@@ -4,6 +4,7 @@
   "plugins": ["import", "babel", "jest", "prettier"],
```

Comment:
NRTTL
Now we get dupplicate when on mainchain 
![image](https://github.com/LiskHQ/lisk-desktop/assets/8784876/291602b0-f077-47d1-8484-ab0b951d8f68)